FileGopherServer.h

```diff
@@ -50,17 +50,18 @@ class FileGopherServer
   struct sockaddr_in server_addr, client_addr;
   string root;
   string socket_host;
+  bool show_hidden_files;
   static void error(const char *msg);
   static bool is_regular_file(const char *path);
   static bool is_text_file(const char *file);
   static char get_file_type(const char *file);
   void send_file_contents(int client_fd, const char *file);
-  int get_contents(char buffer[], int size);
+  int get_contents(char buffer[], int size,bool show_hidden_files);
```

Comment:
Since the function is member of the class, it can access `show_hidden_files`.
You don't need to pass it as param.

Reply:
Done